### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by dionyziz**

### CODE REVIEW

File: docs/getting_started.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+all sorts of skills:
+
+ * ***Artificial intelligence***: Improve your game by improving your strategy
+   heuristics.
+ * ***Machine learning***: Use learning techniques to make your programs learn from
+   past games. You can use historical games as your training set.
+ * ***Programming languages***: As the platform is not restrictive about which
+   programming language you can use, you can develop domain-specific languages
+   to tackle particular problems in more extensible ways.
+ * ***Big data***: You can download historical games into your own database and
```
Comment (vitsalis) [inline docs/getting_started.md]: we don't have this feature yet

File: docs/getting_started.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+[endofcodes.com](http://endofcodes.com). Then go to your settings page and
+change your bot URL to the one you've created. You're ready to play! Games take
+place every day at 18:00 (GMT+2).
+
+Modifying your bot
+------------------
+The sample bot is quite stupid. The basic game logic is written in `round.php`,
+which you can edit. The bot behaves as follows:
+
+ * If a creature can attack, make it attack
```
Comment (bugos) [inline docs/getting_started.md]: dot at the end

File: docs/getting_started.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+games.
+
+What's special
+--------------
+If you've participated in other contests, End of Codes is somewhat unique in the
+following ways:
+
+ * All players play against all others in every game. It's not a one-against-one
+   match.
+ * You can use any programming language you like, even your own. There are no
```
Comment (pkakelas) [inline docs/getting_started.md:16]: Maybe here you should add a parenthesis that inform users about the languages in which we have libraries at the moment.

File: docs/getting_started.md [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+
+Creating your bot
+-----------------
+We'll show you how to create a bot in PHP. We're currently developing libraries
+for C/C++ and Python to make it easier for players to play. If you plan to use C
+or C++, you can use cgi behind a web server. If you use python, you can use
+bottle. If you use ruby, you can use sinatra or rails.
+
+We're also planning to develop a protocol that eliminates the need for players
+to port forward. However, for now, you must either enable port forwarding on
```
Comment (pkakelas) [inline docs/getting_started.md:106]: We could have a reference to a tutorial for port fordwarding here.

File: docs/getting_started.md [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+URL must be accessible from the Internet. For example, it can be
+`http://example.com/endofcodes/mybot/bot`. Make sure that visiting this URL
+shows the expected JSON data and returns a 200 HTTP OK response.
+
+Configuring your account
+------------------------
+Now, make sure you create an account on
+[endofcodes.com](http://endofcodes.com). Then go to your settings page and
+change your bot URL to the one you've created. You're ready to play! Games take
+place every day at 18:00 (GMT+2).
```
Comment (pkakelas) [inline docs/getting_started.md:136]: Why don't we tell the time in GMT time?

Comment (dionyziz) [inline docs/getting_started.md:136]: Our target users are in Greece, so it makes sense to give the time in Greek time for now.

**Pull request merged**